Client/Ecosia/Extensions/AppSettingsTableViewController+Ecosia.swift

```diff
@@ -157,9 +157,21 @@ extension AppSettingsTableViewController {
             ResetSearchCount(settings: self),
             EngagementServiceIdentifierSetting(settings: self),
             FasterInactiveTabs(settings: self, settingsDelegate: self),
-            UnleashOnboardingCardNTPSetting(settings: self),
         ]
         
+        if OnboardingCardNTPExperiment.isEnabled {
+            hiddenDebugSettings.append(UnleashOnboardingCardNTPSetting(settings: self))
+        }
+        
+        if SeedCounterNTPExperiment.isEnabled {
```

Comment:
Given `SeedCounterNTPExperiment` also has variants, I guess it's also helpful for debugging to add an `UnleashVariantResetSetting` for it?